src/components/MessagesList/MessagesList.vue

```diff
@@ -424,8 +439,11 @@ export default {
 
 		/**
 		 * Creates a long polling request for a new message.
+		 * @param {boolean} scrollToBottom Whether we should try to automatically scroll to the bottom
 		 */
-		async getNewMessages() {
+		async getNewMessages(scrollToBottom) {
+			scrollToBottom = scrollToBottom === undefined ? true : scrollToBottom
```

Comment:
@nickvergessen note for the future: you can use default values for arguments in ES6 :-)

I'll adjust after rebase